README.md

```diff
@@ -1,231 +1,50 @@
-This directory contains a set of scripts to generate a diagnostic tarball for DSE, DDAC &
-open source Cassandra installations, similar (and partially compatible) to diagnostic
-tarball generated by OpsCenter.
+# DataStax Diagnostic Collector for Apache Cassandra and DSE
 
-Generation of diagnostic tarball for DSE/Cassandra consists of 2 parts:
-1. Collection of data on individual nodes
-2. Merging all data into single file
+A script for collecting a diagnostic snapshot from each node in a Cassandra based cluster.
 
-These 2 steps could be separate because administrators may have different ways to access
-nodes in the cluster, and transferring the data.  The `collect_diag.sh` performs data
-collection from cluster as one command, executed on one of the nodes of the cluster, or
-from Mac or Linux machine outside of the cluster.
+The code for the collector script is in the _ds-collector/_ directory. It must first be built into a collector tarball.
 
-## collect_diag.sh
+This collector tarball is then extracted onto a bastion or jumpbox that has access to the nodes in the cluster. Once extracted, the configuration file (collector.conf) can be edited to match any cluster deployment customisations (e.g. non-default port numbers, non-default log location, etc). The ds-collector script can then be executed; first in test mode and then in collection mode.
 
-This script copies the `collect_node_diag.sh` file to all nodes, executing it in parallel,
-copying back collected data, and generate resulting tarball with DSE Insights or
-diagnostic data using [`generate_diag.sh`](#merging-all-diagnostics-into-single-tarball).
+# Building the Collector
 
-Usage:
+Build the collector using the following make command syntax. You will need Makefile and Docker.
```

Comment:
"You will need Makefile" -> "You will need make"
(The Makefile comes with the collector, they need the make utility instead).